### ISSUE TITLE
Maven verify compiled and build the jar again

### ISSUE THREAD

**Issue by guptadeepak0687** [serenity-bdd/serenity-core#2867]
Whenever i triggered the serenity BDD test cases using mvn verify command. I noticed it always start from compilation, build and then run the test cases, why compilation and building is required again and again?
Can't we build once and use multiple times.?

**Comment by guptadeepak0687** [comment]
Did anyone get a chance to look at this?

**Comment by wakaleo** [comment]
That's just the way maven works (it's not related to Serenity).

**Comment by guptadeepak0687** [comment]
Thanks John, that triggers another point, is their any way to run serenity without maven?

**Comment by wakaleo** [comment]
No, it's a test library designed to be run as part of a build process.

**Comment by guptadeepak0687** [comment]
Thanks John, Here is one need, that my team wants serenity scripts  to run without maven. so if you could suggest anything would be helpful for me.

**Comment by wakaleo** [comment]
That's not really supported at the moment.

**Comment by globalworming** [comment]
may I ask why @guptadeepak0687? "without maven".. where is that requirement coming from? do you need the whole serenity feature set but without maven? or are you fine with less?
